src/cljs/org/broadinstitute/firecloud_ui/common/icons.cljs

```diff
@@ -77,5 +80,4 @@
    (fn [{:keys [props]}]
      [:span {:style {:display "inline-block" :position "relative" :verticalAlign "middle"
                      :height (:size props) :width (:size props)}}
-      (style/center {}
-        (font-icon {:style {:color "#fff" :fontSize (:size props)}} :status-warning))])})
+      (style/center {} [:span {:style {:color "white" :fontSize (:size props) :fontWeight "500"}} "!"])])})
```

Comment:
There wasn't a bare exclamation mark, but there are two versions of one in a circle.  Actually, one of them is used for `:error`.  We could use that.